src/CHANGES.txt

```diff
@@ -99,6 +99,12 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Docbook builder provides a fallback if lxml fails to generate
       a document with tostring().
 
+  From James Benton:
+    - Improve Visual Studio solution/project generation code to add support
+      for a per-variant cppflags. This is especially useful for passing through
+      the C++ standard version, e.g. /std:c++17, which ensures intellisense
+      uses the correct version of C++.
+
```

Comment:
Please list all flags affected..

Reply:
I don't know the full list of flags which affect intellisense. If it is preferred we could allow only /std:c++N through as that seems to be the most important one - currently if you open a project using C++17 features it is full of red squiggly lines!

Reply:
Note that most build environments will not set /Zc:__cplusplus, however intellisense will not recognize /std:c++N without /Zc:__cplusplus being set. This is why in the project generation I scan for /std:c++N and add /Zc:__cplusplus if it is not present.

Reply:
Right. So please add that to CHANGES.txt.  Otherwise people will have to look at the source code to figure out which version this changed in.  The idea is that CHANGES.txt should give enough information to know user facing details of what changed. This detail would be one of them.  Once you add this I should be able to merge.